cache/in-memory/src/event/guild.rs

```diff
@@ -174,38 +174,38 @@ impl UpdateCache for GuildDelete {
 }
 
 impl UpdateCache for GuildUpdate {
-    fn update(&self, cache: &InMemoryCache) {
+    fn update(self, cache: &InMemoryCache) {
         if !cache.wants(ResourceType::GUILD) {
             return;
         }
 
         if let Some(mut guild) = cache.guilds.get_mut(&self.0.id) {
             guild.afk_channel_id = self.afk_channel_id;
             guild.afk_timeout = self.afk_timeout;
-            guild.banner = self.banner;
-            guild.default_message_notifications = self.default_message_notifications;
-            guild.description = self.description.clone();
-            guild.features = self.features.clone();
-            guild.icon = self.icon;
-            guild.max_members = self.max_members;
-            guild.max_presences = Some(self.max_presences.unwrap_or(25000));
-            guild.mfa_level = self.mfa_level;
-            guild.name = self.name.clone();
-            guild.nsfw_level = self.nsfw_level;
-            guild.owner = self.owner;
-            guild.owner_id = self.owner_id;
-            guild.permissions = self.permissions;
-            guild.preferred_locale = self.preferred_locale.clone();
-            guild.premium_tier = self.premium_tier;
+            guild.banner = self.0.banner;
+            guild.default_message_notifications = self.0.default_message_notifications;
```

Comment:
The old version used `Deref` and `Copy` to access the owned version, but a new style is now introduced for `Copy` items. For consistency's sake please use `.0` for every field or only for non `Copy` types.